src/components/assets/styles/account.scss

```diff
@@ -1,5 +1,10 @@
 @import 'src/css/quasar.variables.scss';
 
+.wrapper--account {
```

Comment:
Previously, we used tailwindcss to quickly bootstrap a project without a dedicated designer, but since we now have someone, we can stop using tailwind from now on, especially if the project configuration is a hassle.

Reply:
I don't want to rely on tailwindcss too much as long as sass can handle the styling because I think styling and HTML should be not mixed. I want to make the `.vue` file as simple as possible.

Reply:
Then I think we can remove tailwind from the project. @sirius651 what do you think?

Reply:
agreed with simplicity of the components, but what if we just separate the contents of the class if too-much mixing is a problem? and if we go `scss`, it would be good to define the policy of `scss`. for example, what are the rules for making classNames like `row__right`, `column--balance`? how to maintain reusable global classes instead of boilerplate parts? In this point, `tailwind` can take care of it I think

Reply:
@sirius651 
I've applied the css class name by [BEM](https://css-tricks.com/bem-101/) methodology.
https://github.com/AstarNetwork/astar-apps/pull/279#issue-1167205769

Reply:
I see, in that case, I also should follow that rule